include/kvdk/engine.h

```diff
@@ -34,6 +34,7 @@ typedef struct KVDKWriteBatch KVDKWriteBatch;
 typedef struct KVDKIterator KVDKIterator;
 
 typedef enum { SORTED, HASH } KVDKIterType;
+typedef enum {LEFT, RIGHT} KVDKListOpType
```

Comment:
This thing should not be exposed to user. It's a internal thing.

Reply:
It 's my old code, I forgot deleting it. Thanks for reminder.